### ISSUE TITLE
TypeError: get_losses() missing 1 required positional argument: 'origin_preds'

### ISSUE THREAD

**Issue by zxccsssd** [remaro-network/KD-YOLOX-ViT#1]
When I launched the Student training,I was faced with this problem.

**Comment by martinaub** [comment]
Hi @zxccsssd, thank you for your comment. Could you share a screenshot of your parameters set-up in the student file, which I suppose should be exps/default/yolox_nano.py? Thank you :)

**Comment by zxccsssd** [comment]
Sorry, I didn't reply in time.I plan to use yolox_tiny，is this supported？

**Comment by martinaub** [comment]
Yes, the yolox_tiny is supported and should work as expected. Ensure that the dataset path is correct and the parameters in both student and teacher files are correctly set.
Let me know if you get an error while launching it.
